packages/framer-motion/src/render/utils/get-variant-context.ts

```diff
@@ -0,0 +1,43 @@
+import { VisualElement } from "../VisualElement"
```

Comment:
This is a bit of a drive-by moving the method out into a function. To improve code-splitting I want to prefer functions over methods where possible.